sensitivity-api/src/main/java/com/powsybl/sensitivity/SensitivityComputation.java

```diff
@@ -28,12 +29,24 @@
  */
 public interface SensitivityComputation extends Versionable {
     /**
-     * Run an asynchronous sensitivity computation job using given parameters and input provider
+     * Run an asynchronous systematic sensitivity computation job using given parameters and input provider
      *
      * @param factorsProvider sensitivity factors provider for the computation
+     * @param contingenciesProvider contingencies provider for the computation
      * @param workingStateId id of the network base state for the computation
      * @param sensiParameters sensitivity computation parameters
-     * @return the sensitivity computation results
+     * @return the sensitivity computation results in N and N-1
+     */
+    CompletableFuture<SensitivityComputationResults> run(SensitivityFactorsProvider factorsProvider, ContingenciesProvider contingenciesProvider, String workingStateId, SensitivityComputationParameters sensiParameters);
```

Comment:
add a default implementation to prevent breaking change here

Reply:
I had thought of handling that on the implementation side, is it wrong?